.github/workflows/publish.yml

```diff
@@ -52,7 +52,9 @@ jobs:
             Source commit: ${{ github.sha }}
 
       - name: '❄ Install dependencies'
-        run: 'nix develop .#docs --command emanote --version'
+        run: |
+          nix profile install github:srid/emanote
```

Comment:
That seems to fix the problem, but it makes job time longer (from 3 mins to 8 mins)... Not sure if there's a better way to go.

Reply:
I'm not sure what was happening before, but updating the `emanote` flake input with:

```
nix flake lock --update-input emanote
```

allowed me to go back to the old way of including `emanote` in the shell.

The first run of this is likely to be slow but should be quicker once cached.

Reply:
[First run](https://github.com/input-output-hk/cardano-wallet/runs/7085001893): ~8m
[Second run](https://github.com/input-output-hk/cardano-wallet/runs/7085100813): ~4m